README.md

```diff
@@ -1,4 +1,4 @@
-sonar
+sonar [![Build Status](https://secure.travis-ci.org/mrcrgl/sonar.png)](http://travis-ci.org/mrcrgl/sonar)
```

Comment:
This looks like it is pointing to your fork and thus seems inappropriate to merge.